app/src/main/java/com/duckduckgo/app/browser/BrowserTabViewModel.kt

```diff
@@ -2446,6 +2449,13 @@ class BrowserTabViewModel @Inject constructor(
         }
     }
 
+    fun onNewTabFavouritesShown() {
+        Timber.d("Sync-Favourites: New Tab screen opened, triggering sync")
```

Comment:
NIT: I assume we can remove the comment since we are already logging trigger events inside engine.